pom.xml

```diff
@@ -457,6 +458,12 @@
         <artifactId>chill-java</artifactId>
         <version>${chill.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.github.jnr</groupId>
+        <artifactId>jnr-posix</artifactId>
+        <version>${jnr.version}</version>
```

Comment:
You're welcome to just the version here if it's used one place, for simplicity, no big deal

Reply:
Inlined it.
If/when we relook at using jnr more broadly - we might need to pull it out (since there are a bunch of artifacts).